module-db/Interface/NotificationsRecord.cpp

```diff
@@ -243,3 +255,22 @@ bool NotificationsRecordInterface::processIncrement(NotificationsRecord::Key key
 
     return ret;
 }
+
+bool NotificationsRecordInterface::processDecrement(NotificationsRecord::Key key, size_t delta)
+{
+    auto ret = false;
+
+    if (auto record = GetByKey(key); record.isValid() && record.value > 0) {
+        if ((ssize_t(record.value) - delta <= 0)) {
```

Comment:
why cast to unsigned? :) doesn't seem like it's needed?

Reply:
Fixed as discussed.